### DISCUSSION TITLE
un-official version on Play store (seal)

### DISCUSSION THREAD

**Discussion by acdel123** [gsantner/markor#2087] [General] [answered]
@gsantner 

Today I stumbled upon what appears to me to be someone who has taken the Markor app and uploaded it to the play store  (see https://play.google.com/store/apps/details?id=com.seal.markor)

I don't know enough about the legalities of open source software. But even if this isn't strictly illegal, I thought you'd want to know about it if you weren't already aware.

**Comment by gsantner** [comment]
I got quite a few mails already. For sure, Seal is not by me/"""project official""" if you may call so.

**Comment by gsantner** [comment, marked as answer]
See #2099
